### PR TITLE
Support list type flow input with image data

### PR DESCRIPTION
**Pull request by YingChen1996**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: src/promptflow/promptflow/_sdk/data/executable/main.py.jinja2 [hunk 1]
```diff
@@ -2,32 +2,66 @@ import json
...
 import streamlit as st
 from pathlib import Path
+from streamlit_quill import st_quill
+from bs4 import BeautifulSoup, NavigableString, Tag
 
 from promptflow._sdk._utils import print_yellow_warning
 from promptflow._sdk._serving.flow_invoker import FlowInvoker
 
-
+IMAGE_KEY = "data:image/png:base64"
```
Comment (brynn-code) [inline src/promptflow/promptflow/_sdk/data/executable/main.py.jinja2:11]: data:image/png;base64, should be ; instead of :

Comment (YingChen1996) [reply src/promptflow/promptflow/_sdk/data/executable/main.py.jinja2]: fixed

File: src/promptflow/promptflow/_sdk/data/executable/main.py.jinja2 [hunk 2]
```diff
@@ -41,34 +88,76 @@ def start():
...
+{{ ' ' * indent_level * 3 }}file_contents = {{flow_input}}.read()
+{{ ' ' * indent_level * 3 }}{{flow_input}} = base64.b64encode(file_contents).decode('utf-8')
+{% endif %}
+{% endfor %}
+
+            submit({{flow_inputs_params}})
 
-    if clear_bt:
-        clear_chat()
+        if clear_bt:
```
Comment (brynn-code) [inline src/promptflow/promptflow/_sdk/data/executable/main.py.jinja2:159]: extra indent?

Comment (YingChen1996) [reply src/promptflow/promptflow/_sdk/data/executable/main.py.jinja2:216]: updated

File: src/promptflow/promptflow/_sdk/_serving/flow_invoker.py [hunk 3]
```diff
@@ -10,13 +11,17 @@
...
 from promptflow._sdk._utils import (
+    dump_flow_result,
     get_local_connections_from_executable,
     override_connection_config_with_environment_variable,
     resolve_connections_environment_variable_reference,
     update_environment_variables_with_connections,
 )
 from promptflow._sdk.entities._connection import _Connection
+from promptflow._sdk.operations._flow_operations import FlowOperations
+from promptflow._utils.multimedia_utils import convert_multimedia_date_to_base64, persist_multimedia_date
```
Comment (lalala123123) [inline src/promptflow/promptflow/_sdk/_serving/flow_invoker.py:22]: typo: persist_multimedia_data

**Review by brynn-code — APPROVED** [review]

**Review by lalala123123 — APPROVED** [review]

**Pull request merged**